doc/release/1.14.0-notes.rst

```diff
@@ -40,6 +40,17 @@ New Features
 Improvements
 ============
 
+``void`` datatype elements are now printed in hex notation
+----------------------------------------------------------
+A hex representation compatible with the python ``bytes`` type is now printed
+for unstructured ``np.void`` elements (eg, ``V4`` datatype). Previously, in
+python2 the raw void data of the element was printed to stdout, or in python3
+the integer byte values were shown.
+
+printing style for ``void`` datatypes is now customizable
+---------------------------------------------------------
+The printing style of ``np.void`` arrays is customizable using the
+``formatter`` argument to ``np.set_printoptions``, using the ``'void'`` key.
```

Comment:
the void type was always customizable, but before it was done through the (still) mis-documented 'numpystr' key